slatedb/src/garbage_collector.rs

```diff
@@ -30,7 +31,14 @@ trait GcTask {
     fn resource(&self) -> &str;
     fn interval(&self) -> Duration;
     fn ticker(&self) -> Interval;
-    async fn collect(&self, now: DateTime<Utc>) -> Result<(), SlateDBError>;
+    async fn collect(&self, now_in_millis: i64) -> Result<(), SlateDBError>;
+}
+
+pub(crate) fn gc_task_time(millis: i64) -> DateTime<Utc> {
```

Comment:
Yea, I think you're right. I don't see why we're using `chrono` here. IMO, it'd be better to remove it.